packages/experimental/Menu/src/MenuItemCheckbox/useMenuItemCheckbox.ts

```diff
@@ -11,7 +11,7 @@ export const useMenuItemCheckbox = (props: MenuItemCheckboxProps): MenuItemCheck
   const { onClick, accessibilityState, componentRef = defaultComponentRef, disabled, ...rest } = props;
   const pressable = useAsPressable({ ...rest, disabled, onPress: onClick });
   const onKeyProps = useKeyProps(onClick, ' ', 'Enter');
-  const hasSubmenu = useMenuContext().isSubmenu;
```

Comment:
Feels like this shouldn't have gone away?

Reply:
No this is by design. The MenuItemCheckbox doesn't support submenus. (Doesn't really make sense for a submenu trigger to be a toggle-able button)